llvm/lib/Transforms/Instrumentation/AllocToken.cpp

```diff
@@ -357,11 +380,19 @@ bool AllocToken::instrumentFunction(Function &F) {
   }
 
   bool Modified = false;
-  for (auto &[CB, Func] : AllocCalls)
-    Modified |= replaceAllocationCall(CB, Func, ORE, TLI);
 
-  if (Modified)
-    NumFunctionsInstrumented++;
+  if (!AllocCalls.empty()) {
+    for (auto &[CB, Func] : AllocCalls)
+      Modified |= replaceAllocationCall(CB, Func, ORE, TLI);
+    if (Modified)
+      NumFunctionsInstrumented++;
+  }
+
+  for (auto *II : IntrinsicInsts) {
+    replaceIntrinsicInst(II, ORE);
+    Modified = true;
```

Comment:
We don't count functions that only had intrinsics in `NumFunctionsInstrumented`. Is that intentional? If so, please document in a comment why.